src/randomPicker.ts

```diff
@@ -101,4 +141,19 @@ export class RandomPicker<T> {
     this.items = items;
     this.updateTotalWeight();
   }
+
+  private indexToPick(items = this.items): number {
```

Comment:
This method is duplicating the logic in `pick` (the for loop basically). I don't think we need it.

Reply:
Understood will make changes accordingly